space_view3d_virtual_reality/operator.py

```diff
@@ -249,7 +250,8 @@ def init(self, context):
         self._hash_master = hash(context.area)
 
         # setup modal
-        self._timer = wm.event_timer_add(1.0 / 75.0, context.window) # 75 Hz
+        # TODO: check: cedeon changed to: 90Hz
+        self._timer = wm.event_timer_add(1.0 / 90.0, context.window) # 90 Hz
```

Comment:
This may need to be an option per HMD. I will need to check how does the Oculus behave with this change.